iothub/device/src/Transport/Http/HttpClientHelper.cs

```diff
@@ -59,6 +75,9 @@ internal sealed class HttpClientHelper : IHttpClientHelper
             _authenticationHeaderProvider = authenticationHeaderProvider;
             _defaultErrorMapping = new ReadOnlyDictionary<HttpStatusCode, Func<HttpResponseMessage, Task<Exception>>>(defaultErrorMapping);
 
+            ServicePoint servicePoint = ServicePointManager.FindServicePoint(_baseAddress);
+            servicePoint.ConnectionLeaseTimeout = s_defaultConnectionLeaseTimeout.Milliseconds;
```

Comment:
Any advantage in making this configurable?

Reply:
It isn't configurable anywhere else in the SDK yet, but we could add that feature later